src/webrtc/call.ts

```diff
@@ -2516,18 +2514,7 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
                     return;
                 }
 
-                await this.client.encryptAndSendToDevices(
-                    [
-                        {
-                            userId,
-                            deviceInfo: this.opponentDeviceInfo,
-                        },
-                    ],
-                    {
-                        type: eventType,
-                        content,
-                    },
-                );
+                // Here we were calling `MatrixClient.encryptAndSendToDevices` which is not supported by the rust cryptography.
```

Comment:
Could we replace this with the recommended encryptToDeviceMessages + queueToDevice calls rather than breaking it?

Reply:
Okay, well, perhaps the comment could express a more clearly that this area is a TODO if we ever tried to use it again? I'm thinking like:

```
// TODO: Here we were sending the event to the opponent's device as a to-device message with MatrixClient.encryptAndSendToDevices.
// However due to the switch to Rust cryptography we need to migrate to the new encryptToDeviceMessages API.
throw new Error("Unimplemented");
```